docs/source/help.rst

```diff
@@ -6,21 +6,21 @@ AboutCode is a suite of open source projects.
 AboutCode Projects
 ******************
 
-- **[ScanCode Toolkit](https://github.com/nexB/scancode-toolkit)**: This is a set of code scanning tools to detect the origin and license of code and dependencies. ScanCode Toolkit uses a plug-in architecture to run a series of scan-related tools in one process flow. This is the most popular project and is used by hundreds of software teams. https://github.com/nexB/scancode-toolkit . The lead maintainer is @pombredanne
+- `ScanCode Toolkit <https://github.com/nexB/scancode-toolkit>`_: This is a set of code scanning tools to detect the origin and license of code and dependencies. ScanCode Toolkit uses a plug-in architecture to run a series of scan-related tools in one process flow. This is the most popular project and is used by hundreds of software teams. https://github.com/nexB/scancode-toolkit . The lead maintainer is @pombredanne
```

Comment:
Can you elaborate on the reason for these markup changes? what is the result from this visually?

Reply:
Specifically I meant the use of backticks

Reply:
@pombredanne Notice on this page (the current version of this file): https://aboutcode.readthedocs.io/en/latest/help.html

Reply:
Perfect. Thank you for the details.
And will these still be bolded?

Reply:
No, there is currently no way to bold links like this in RST... See: http://docutils.sourceforge.net/FAQ.html#is-nested-inline-markup-possible